eng/build.py

```diff
@@ -89,6 +89,21 @@ def _run_tests(exit_immediately: bool = True):
     )
 
 
+def _run_integration_tests(exit_immediately: bool = True):
+    _run_command(
+        [
+            "poetry",
+            "run",
+            "pytest",
+            "tests_integration_sql",
+            "--useintegratedsecurity=false",
+            "--username=sa",
+            "--password=abcdefgh1!",
```

Comment:
Hardcoded here rather than redo build.py to accept additional parameters - such as password.

Reply:
Suggestion: rename to `_run_integration_tests_on_github` or something like that, and maybe add a comment above this line mentioning why it is safe to have a hard-coded password. Might help with controlling perception. I agree that this is a safe and good thing to do. Just want to further head off any concern from casual readers (or from security auditors).